lld/test/COFF/pdb-from-outputpath.test

```diff
@@ -0,0 +1,34 @@
+Replicate this scenario:
+
+$ cat a.c
+struct Foo { int x; };
+int g(struct Foo *p);
+int main() {
+  struct Foo f = {42};
+  return g(&f);
+}
+
+$ cat b.c
+struct Foo { int x; };
+int g(struct Foo *p) { return p->x; }
+
+$ cl -c a.c b.c -Zi -Fdts.pdb
+
+$ lld-link a.obj b.obj -debug -entry:main -nodefaultlib -out:t.exe
+
+RUN: rm -rf %t && mkdir -p %t && mkdir -p %t/lib && cd %t
+
+RUN: yaml2obj %S/Inputs/pdb-type-server-simple-a.yaml -o lib/a.obj
+RUN: yaml2obj %S/Inputs/pdb-type-server-simple-b.yaml -o lib/b.obj
+RUN: llvm-pdbutil yaml2pdb %S/Inputs/pdb-type-server-simple-ts.yaml -pdb ts.pdb
+
+RUN: lld-link lib/a.obj lib/b.obj -debug -entry:main -nodefaultlib -out:t.exe
```

Comment:
Can you rather extend the other test, `pdb-type-server-simple.test`? I think you could just `cp` the files where you need them in the `%t` folder. Also what you want to check here is the absence of warnings, `FileCheck -check-prefix=OUTPUT` and then `OUTPUT-NOT: warning`. The point is that you should run the test without your fix, and make sure it fails first. Then it should suceed with your fix applied.

Reply:
> The point is that you should run the test without your fix, and make sure it fails first. Then it should suceed with your fix applied.

Oh, so how to run two different lld-links in the test code

Reply:
What I mean you should revert your code changes from your local git branch, and only keep the test. Compil & run, the test should fail. Then unstash or reapply your code changes, compil & run, the test should now pass.

Reply:
Move to `pdb-type-server-simple.test`, use `yaml2obj` to generate an additional pdb, which can be modified to `cp` if necessary

Reply:
Can you rather leave the existing test unmodified and add another block after to run your test? I think you could create a %t2 folder and %t2/lib and cp the .obj and the .pdb files from the previous step? Then add another line with `lld-link ...` and `FileCheck -check-prefix=OUTPUT` and then `OUTPUT-NOT: failed to load reference`. You can take inspiration from other tests, such as `pdb-natvis.test`. Except that in your case you could pack all these steps together (mkdir(s), cp, lld-link and the OUTPUT line)

Reply:
As suggested by comments, I made changes and reviewed again